db-init-action/action.yml

```diff
@@ -0,0 +1,50 @@
+name: "Initialize Commercial Database"
+Description: Initialize Commercial Database in parsley-services instance"
+inputs:
+  db-instance:
+    description: 'Database instance name'
+    required: true
+  db-name:
+    description: 'Database name'
+    required: true
+  gcp-workload-id-service-account-email:
+    description: "GCP service account email to assume"
+    required: true
+  gcp-workload-id-provider:
+    description: "GCP workload ID provider name"
+    required: true
+  gcp-project-id:
+    description: "GCP project of the database"
+    required: true
+  cloudsql-superuser:
+    description: "User created by Terraform in CloudSQL"
+    required: true
+  db-init-script-path:
+    description: "Path to init script relative to repository root"
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+    - uses: actions/checkout@v3
+    
+    - id: 'auth'
+      uses: 'google-github-actions/auth@v1'
+      with:
+        workload_identity_provider: ${{ inputs.gcp-workload-id-provider }}
+        service_account: ${{ inputs.gcp-workload-id-service-account-email }}
+        
+    - uses: wagnerpereira/gce-cloudsql-proxy-action@v2
+      with:
+        instance: ${{ inputs.db-instance }}
+        
+    - id: 'secrets'
```

Comment:
don't need to use an `id` here, also you can ditch the single quotes on `uses`, not necessary